apiserver/pom.xml

```
@@ -114,6 +114,11 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.dependencytrack</groupId>
+            <artifactId>vuln-data-source-csaf</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

The transitive `slf4j-jdk14` dependency seems to conflict with our desired Logback slf4j provider:

```suggestion
            <exclusions>
                <exclusion>
                    <groupId>org.slf4j</groupId>
                    <artifactId>slf4j-jdk14</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```

---

That’s why I got these weird log error messages.